tests/drivers/uart/uart_async_api/src/test_uart_async.c

```diff
@@ -91,28 +118,33 @@ static void *single_read_setup(void)
 
 ZTEST_USER(uart_async_single_read, test_single_read)
 {
-	uint8_t rx_buf[10] = {0};
-
-	/* Check also if sending from read only memory (e.g. flash) works. */
-	static const uint8_t tx_buf[5] = "test\n";
+	DATA_TYPE rx_buf[10] = {0};
+	const uint8_t nr_of_rx_values = sizeof(rx_buf)/sizeof(DATA_TYPE);
```

Comment:
You can use the ARRAY_SIZE() macro and probably just use it in the function to avoid declaring an extra const variable.

Reply:
Fixed